docs/src/pages/docs/events.md

```diff
@@ -0,0 +1,200 @@
+# Events
+
+Events are an incredibly powerful feature in Anchor. Rather than polling or refreshing to
+find out if an account's state has changed, events are like callbacks that save space, compute,
+and enable asynchronous programming.
+
+The main downside to events are that they are base64 encoded, and therefore not human readable.
+Despite this tradeoff, the UI can easily decode them, and the user benefits from this 
+compact mode of logging.
```

Comment:
```suggestion
The main downside to events are that they are base64 encoded, and therefore not human readable.
Despite this tradeoff, the UI can easily decode them, and the user benefits from this 
compact mode of logging. There are many use cases for events, ranging from simple return values, to compact storage storage solutions for data logging.
```

This sets you up nicely to flow into the next section of chapter/page, which I've also suggested a restructure on.